protocol/app/msgs/all_msgs_test.go

```diff
@@ -33,6 +33,7 @@ func TestAllTypeMessages(t *testing.T) {
 
 	// Assert.
 	require.Equal(t, allTypes, lib.GetSortedKeys[sort.StringSlice](msgs.AllTypeMessages))
+	require.Len(t, msgs.AllTypeMessages, 210)
```

Comment:
not the biggest fan of using direct numbers here, i can define some constants if this bothers anyone